landing-page/content/common/how-to-release.md

```diff
@@ -192,11 +212,15 @@ This release includes important changes that I should have summarized here, but
 
 Please download, verify, and test.
 
-Please vote in the next 72 hours.
+Please vote in the next 72 hours. (Weekends excluded)
```

Comment:
I'm not sure whether that's actually true or not. Might be good to have a definitive answer whether this should be `working days` vs `the next 3 days`.
@rdblue I remember you mentioned something around this a while ago, so maybe we could clarify it here

Reply:
Apache guidelines

> Release votes SHOULD remain open for at least 72 hours.

https://www.apache.org/legal/release-policy.html#release-approval

https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions

Reply:
I think it's up to the project to set the length as long as it's at least 3 days. At Impala we found it practical to make this "working days" as people in the community were quite inactive on the weekends for instance and a vote started on Friday usually didn't get enough attention.

Reply:
Should we then change this into working days? I think de-facto is already the case.

Reply:
@Fokko Yeah, that's a good idea, but then there might be confusion with how to count the national holidays that aren't the same in each country :)